Dockerfile

```diff
@@ -1,45 +1,152 @@
 ## This Dockerfile is meant to aid in the building and debugging patroni whilst developing on your local machine
 ## It has all the necessary components to play/debug with a single node appliance, running etcd
-FROM postgres:10
-MAINTAINER Alexander Kukushkin <alexander.kukushkin@zalando.de>
+ARG PG_MAJOR=10
```

Comment:
Let's use 11, should we?

Reply:
It doesn't bring many benefits but unfortunately adds 82MB to uncompressed image and 23MB to compressed version (55MB vs 78MB) due to llvm libraries :(

In any case it is not a big deal to change the default value to 11 and specify the custom version during build: `--build-arg PG_MAJOR=10`